packages/odyssey-storybook/src/components/odyssey-mui/Form/Form.mdx

```diff
@@ -0,0 +1,111 @@
+import { ArgsTable, Canvas, Meta, Story } from "@storybook/addon-docs";
+
+<Meta anchor />
+
+# Form
+
+The `Form` component arranges input controls to help with the layout and building of a form.
+
+## Overview
+
+Forms are used for submitting data so it is important for them to be easy to understand and easy to complete. There is generally not one simple way to create all forms but following basic usability guidelines can provide a good starting point.
+
+<Canvas>
+  <Story id="mui-components-forms-form--simple" />
+</Canvas>
+
+## Content
+
+`Form` supports the following sections:
+
+### Form header
+
+Forms should include a heading that describes the form. This heading may be provided by the form's context; for example, a modal or page's title.
+
+An additional description can follow the heading.
+
+<Canvas>
+  <Story id="mui-components-forms-form--with-header" />
+</Canvas>
+
+### Field groups
+
+Field groups allow you to group fields together within the same form.
+
+Field groups should be labeled by a legend. They can also include descriptions.
+
+<Canvas>
+  <Story id="mui-components-forms-form--with-legend" />
+</Canvas>
+
+### Form actions
+
+This section houses buttons to allow the user to submit, clear, or exit the form.
+
+## Best practices
+
+### Length
+
+In general, it is good for forms to be as simple and short as possible. Eliminating unnecessary fields will reduce user effort and translate to better completion rates.
+
+Often in products like Okta, it will be difficult to avoid more complex forms for things like settings or configurations. In these cases based on context, it is an acceptable solution.
+
+### Layout
+
+The `Form` component is designed in a single-column layout to maintain a logical flow of completion.
+
+Adding multiple columns in a form can generate confusion for users and lead to usability problems such as having more than one way to interpret the form and forcing users to reorient their eyes to different places in the form.
```

Comment:
I like bolding important statements in sentences to make them easier to scan. What do you think?

```suggestion
Adding multiple columns in a form can generate confusion for users and lead to usability problems such as having more than one way to interpret the form and forcing users to **reorient their eyes** to different places in the form.
```

Reply:
With this and the docs question above, Gretchen and Content Design are going to be reviewing, so I'd rather leave this type of stuff up to them.